### PR TITLE
MM-38582 - Simplify status filtering widget for playbook runs

### PR DESCRIPTION
**Pull request by cpoile**
#### Summary
- Simplify the status filter, which fixes a couple bugs we found in the v1.20 fit&finish bug bash 

#### Ticket Link
- https://mattermost.atlassian.net/browse/MM-38582

#### Checklist

- [ ] ~~Telemetry updated~~
- [ ] ~~Gated by experimental feature flag~~
- [x] Unit tests updated

### CODE REVIEW

File: webapp/src/components/backstage/runs_list/filters.tsx [hunk 1]
```diff
@@ -87,10 +87,10 @@ const Filters = ({fetchParams, setFetchParams, fixedTeam}: Props) => {
         });
     };
 
-    const setStatus = (statuses: string[]) => {
+    const setFinishedRuns = (checked?: boolean) => {
+        const statuses = checked ? [PlaybookRunStatus.InProgress, PlaybookRunStatus.Finished] : [PlaybookRunStatus.InProgress];
         setFetchParams((oldParams) => {
-            return {...oldParams, statuses, page: 0}
-            ;
```
Comment (agarciamontoro) [inline webapp/src/components/backstage/runs_list/filters.tsx:93]: I love how the linter was perfectly ok with this :joy:

**Review by agarciamontoro — APPROVED** [review]
Looks perfect, thank you!

File: webapp/src/components/backstage/runs_list/filters.tsx (hunk 1 above)
Comment (calebroseland) [reply webapp/src/components/backstage/runs_list/filters.tsx:93]: Current rules are showing their age!

**Review by calebroseland — APPROVED** [review]
Should these two be swapped. Looks a little odd/asymmetrical 
CleanShot 2021-09-22 at 11 10 02@2x

**Comment by cpoile** [comment]
@calebroseland Good idea, changed to:
image

@abhijit-singh can I get a quick approval before we merge?

**Review by abhijit-singh — APPROVED** [review]

**Pull request merged**